CHANGELOG.md

```diff
@@ -1,19 +1,46 @@
+## 1.8.0
+
+### ✨ Features and improvements
+* Level-of-detail support for tile coverage ([#8975](https://github.com/mapbox/mapbox-gl-js/pull/8975))
+* Update `image` expression SDK support table ([#9228](https://github.com/mapbox/mapbox-gl-js/pull/9228))
+* Reduce size of line atlas by removing unused channels ([#9232](https://github.com/mapbox/mapbox-gl-js/pull/9232))
+* Prevent empty buffers from being created for debug data when unused ([#9237](https://github.com/mapbox/mapbox-gl-js/pull/9237))
+* Add space between distance and unit in scale control ([#9276](https://github.com/mapbox/mapbox-gl-js/pull/9276)) (h/t [gely](https://api.github.com/users/gely))
+* Add circle to GeolocateControl showing accuracy of position ([#9253](https://github.com/mapbox/mapbox-gl-js/pull/9253)) (h/t [Meekohi](https://api.github.com/users/Meekohi))
+
+### 🐞 Bug fixes
+* `line-dasharray` is now ignored correctly when `line-pattern` is set ([#9189](https://github.com/mapbox/mapbox-gl-js/pull/9189))
+* Remove `Object.values` usage that was causing IE11 to fail ([#9193](https://github.com/mapbox/mapbox-gl-js/pull/9193))
+* Fix line distances breaking gradient across tile boundaries ([#9220](https://github.com/mapbox/mapbox-gl-js/pull/9220))
+* Fix a bug where lines with duplicate endpoint could disappear at zoom 18+ ([#9218](https://github.com/mapbox/mapbox-gl-js/pull/9218))
+* Tile requests to Mapbox sources that don't use Mapbox URLs are now billed correctly ([#9217](https://github.com/mapbox/mapbox-gl-js/pull/9217))
+* Ctrl-click to drag rotate the map is disabled if the Alt, Cmd or Windows key is also pressed ([#9203](https://github.com/mapbox/mapbox-gl-js/pull/9203))
+* Pass errors to `getClusterExpansionZoom`, `getClusterChildren`, and `getClusterLeaves` callbacks ([#9251](https://github.com/mapbox/mapbox-gl-js/pull/9251))
+* Fix a rendering performance regression ([#9261](https://github.com/mapbox/mapbox-gl-js/pull/9261))
+* Fix visual artifact for `line-dasharray` ([#9246](https://github.com/mapbox/mapbox-gl-js/pull/9246))
+* Don't try to update the location indicator before the UI is set up ([#9288](https://github.com/mapbox/mapbox-gl-js/pull/9288)) (h/t [Meekohi](https://api.github.com/users/Meekohi))
```

Comment:
This fixes a bug introduced by a PR within this release, so can be skipped from the changelog